tests/integration/cli/test_cli.py

```diff
@@ -30,8 +31,8 @@ def tmp_dir_path():
 class TestToolsOTXCLI:
     @e2e_pytest_component
     def test_otx_find(self):
-        otx_find_testing(otx_dir)
+        otx_find_testing()
 
     @e2e_pytest_component
     def test_otx_build(self, tmp_dir_path):
-        otx_build_testing(tmp_dir_path, otx_dir, build_backbone_args)
+        otx_build_testing(tmp_dir_path, build_backbone_args)
```

Comment:
Do you plan to implement a test code for checking all available backbones?
Actually, the most challenging point of this work is not making errors for our available backbones.

Reply:
It is possible, including the issues you mentioned, the issue of increasing the test time is also likely to occur. Let's discuss later.